lib/matplotlib/tests/test_animation.py

```diff
@@ -117,6 +117,7 @@ def isAvailable(self):
     ('avconv_file', 'movie.mp4'),
     ('imagemagick', 'movie.gif'),
     ('imagemagick_file', 'movie.gif'),
+    ('pillow', 'movie.gif'),
```

Comment:
this seems to be causing issues on MacOS for some reason. Pillow tests in `test_images` have a decorator `@needs_pillow` which lets most of the tests pass.  This test doesn't, so the tests are now failing....  Of course the underlying issue is that pillow is not being installed properly somehow on macOS...